### ISSUE TITLE
`gemstash authorize fetch` does not work.

### ISSUE THREAD

**Issue by chakrit** [rubygems/gemstash#130]
```
# gemstash authorize fetch
Invalid permission 'fetch'
Valid permissions include: push, yank, unyank
# gemstash --version
Gemstash version 1.0.3
```

This is my config file:

```
---
:base_path: "/root/.gemstash"
:cache_type: memory
:db_adapter: sqlite3
:protected_fetch: true
```

This was mentioned in https://github.com/bundler/gemstash/blob/master/docs/gemstash-authorize.1.md , so it should work right?

**Comment by randycoulman** [comment]
It looks like this was added in #94, but that hasn't been released yet according to the CHANGELOG.

**Comment by olleolleolle** [comment]
@chakrit Does that latest release have the same issue for you?

**Comment by dazuma** [comment]
#94 still doesn't seem to have made it into a release. Is there a plan for the next release yet? /cc @smellsblue

**Comment by bronzdoc** [comment]
https://github.com/rubygems/gemstash/pull/94 was released in v1.1.0 if this still a problem let us know
